File: src/main/java/com/soywiz/minecraft/repair/BukkitPlugin.java
package com.soywiz.minecraft.repair;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.bukkit.plugin.java.JavaPlugin;

@SuppressWarnings("unused")
public class BukkitPlugin extends JavaPlugin {

    @Override
    public void onEnable() {
        this.getCommand("repair").setExecutor(new RepairCommand());
    }
}

File: src/main/java/com/soywiz/minecraft/repair/BukkitPlugin.kt
package com.soywiz.minecraft.repair

import org.bukkit.plugin.java.*

@Suppress("unused")
class BukkitPlugin : JavaPlugin() {
    override fun onEnable() {
        getCommand("repair")?.setExecutor(RepairCommand())
        getCommand("where")?.setExecutor(WhereCommand())
    }
}
